cypress/e2e/login.cy.tsx

```diff
@@ -1,10 +1,12 @@
 const buttonText = 'Conectar con una clave secreta';
 const textLiLoginConfirm =
   'Copiar y pegar su clave secreta lo hace vulnerable a accidentes, ataques y estafas que pueden provocar la pérdida de fondos.';
+const secretKeyFail = 'AASDASD0ASDASD0ASDASD0ASDSA0DASDJASDAKS';
```

Comment:
I think this variable has to be `const secretKey` and the value has to be well-mocked.